rush/rush-lib/src/data/RushConfigurationProject.ts

```diff
@@ -97,6 +99,75 @@ export default class RushConfigurationProject {
   }
 
   /**
+   * Generate a temp_module package.json for this project based on an overall configuration
+   */
+  public generateTempModule(rushConfiguration: RushConfiguration): IPackageJson {
```

Comment:
I disagree. Also, this function doesn't do anything with the filesystem. This function simply tells you what the temp_module should look like, for a project, given a global configuration. This allows me to re-run this code & create an "expected" temp_module when I do the check.

Also, this is more testable than what it was before (this was just checked into a large function). In fact, we should probably add tests for it now that is has been moved.